rllib/policy/eager_tf_policy.py

```diff
@@ -636,7 +637,8 @@ def _stats(self, outputs, samples, grads):
                 })
             return fetches
 
-        def _initialize_loss_with_dummy_batch(self):
+        @override(Policy)
+        def _initialize_loss_dynamically(self):
```

Comment:
Why the name change? The previous one seemed more clear.

Reply:
done